app/test/login_testing.go

```diff
@@ -0,0 +1,264 @@
+package test
```

Comment:
app/test package should not be committed. Previously deleted in https://github.com/almighty/almighty-core/commit/47da9269b14b3fb35c9544ca543b0c7a103b070e and should be in .gitignore